src/corehost/cli/deps_format.cpp

```diff
@@ -122,34 +122,30 @@ void deps_json_t::reconcile_libraries_with_targets(
     }
 }
 
-pal::string_t get_own_rid()
+// Returns the RID for the platform the host is running on.
+pal::string_t get_current_rid()
```

Comment:
Should this method also respect the environment variable `"DOTNET_RUNTIME_ID"`?  The managed code looks to that first.